lib/XMLHttpRequest.js

```diff
@@ -328,12 +329,14 @@ function XMLHttpRequest(opts) {
           } else {
             self.status = 200;
             self.responseText = data;
+            self.response = Buffer.from(fs.readFileSync(unescape(url.pathname), 'binary'), 'binary');
```

Comment:
I don't think this does what you want, `binary` encoding in Node.js is an alias for `latin1`. Also, first encoding the file buffer to `binary` only to then decode it again seems quite silly.